game_patch/multi/commands.cpp

```diff
@@ -42,6 +44,19 @@ void load_prev_level()
     }
 }
 
+void load_rand_level()
+{
+    std::srand(static_cast<unsigned int>(std::time(nullptr)));
```

Comment:
`srand` initializes pseudorandom generator in C standard library. It is supposed to be called once, not every time you generate a random number. Dash already uses `rand` in some places, but I don't see `srand` call, so I guess it just uses some less optimal seed for initialization (e.g. zero). Also not sure how RF affects it (it may call `srand` on it's own but I wouldn't expect it to have effect on GCC rng)